kuma/javascript/src/footer.jsx

```diff
@@ -0,0 +1,118 @@
+//@flow
+import * as React from 'react';
+import GithubIcon from './icons/social/github.svg';
+import TwitterIcon from './icons/social/twitter.svg';
+import InstagramIcon from './icons/social/instagram.svg';
+
+import { getLocale, gettext } from './l10n.js';
+
+export default function Footer() {
+    const locale = getLocale();
+
+    return (
+        <footer id="nav-footer" className="nav-footer">
+            <div className="center">
+                <a href={`/${locale}/`} className="nav-footer-logo">
+                    {gettext('MDN Web Docs')}
+                </a>
+                <div className="footer-group footer-group-mdn">
+                    <h2 className="footer-title">MDN</h2>
+                    <ul className="footer-list">
+                        <li>
+                            <a href="{{ wiki_url('Web') }}">
+                                {gettext('Web Technologies')}
+                            </a>
+                        </li>
+                        <li>
+                            <a href="{{ wiki_url('Learn') }}">
+                                {gettext('Learn Web Development')}
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/docs/MDN/About">{gettext('About MDN')}</a>
+                        </li>
+                        <li>
+                            <a href="{{ wiki_url('MDN/Feedback') }}">
```

Comment:
I don't think this would work in a stable way. It might be replaced for SSR, but as soon as the component rerenders, we'd have this jinja tag as the href value.

Reply:
Whoops! My bad, overlooked.